test/experimental/new_embed_test.html

```diff
@@ -135,13 +135,11 @@
             </div>
         </div>
         <div id="console-output-footer" class="footer-top-border" hidden>
-            <div class="console-output-header">
+            <div id="console-output-header" class="console-output-header">
```

Comment:
nit: An ID and a CSS class with the same name have potential for annoyance later on. Given that the class was probably only ever a single instance, what do you think about changing the CSS rule to apply to the ID instead?

Reply:
agree! fixed.